validphys2/src/validphys/pdfbases.py

```diff
@@ -34,6 +34,23 @@
         (21 , r"g"),
     ))
 
+PIDS_DICT = {
```

Comment:
Why not just use the existing `PDG_PARTONS`?

Reply:
Because they are not really the same, right? `PDG_PARTONS` is for latex. I don't know if would be worth to change the class `LhapdfLike` just to make it work with the latex strings.

Reply:
Did I have this wrong? P.S. remember also @AleCandido's comment.